release-notes.txt

```diff
@@ -1,3 +1,767 @@
+
+System Libraries
+----------------
+
++ sys/event: Add documentation and enforce waiter on wait (#17952)
+  sha256sum shell commands (#18040)
++ sys/shell: new module shell_lock (#13082)
+* Fido2: Enhance flash persistence (#17096)
+* pkg/fatfs, shell: enable RTC support if rtt_rtc is used (#17912)
+* pm: don't (un)block IDLE mode (#17974)
+* sys/posix/include/arpa: use `__restrict` instead of `restrict` (#18034)
+* sys/quad_math: remove unused module for quad arithmetic (#18032)
+* sys/shell: drop `_builtin_cmds` define (#18075)
+* sys/shell_commands: convert to SHELL_COMMAND() (#18152)
+* sys/ztimer: no log for stdio_rtt/semihosting (#18176)
+
+### SUIT
+
++ suit/transport/vfs: add VFS as source for firmware updates (#18045)
++ sys/sc_suit: add seq_no command (#17941)
+* examples/suit_update/test: use 'suit seq_no' to get version (#17973)
+* sys/suit/storage/vfs: initial import (#17943)
+* sys/suit/storage: change log to LOG_DEBUG (#18020)
+* sys/suit: avoid installing payload twice (#17984)
+* sys/suit: save seq_number when parsing manifest (#17939)
+
+### VFS
+
++ sys/shell/vfs: add umount, rmount and mkdir commands (#18112)
++ sys/vfs: add vfs_unmount_by_path() (#18109)
++ sys/vfs_util: add VFS helper functions (#18038)
++ sys/vfs_util: add vfs_file_<hash>() functions and md5sum, sha1sum and
+
+### Rust
+
++ Add some Rust library building infrastructure (#16833)
+* makefiles/rust: Err early on setup issues, provide suggestions (#18220)
+* rust: Build from git branches (#18151)
+* rust: Explicitly select nightly (#18243)
+* rust: Update versions (#18181)
+
+Networking
+----------
+
++ congure_reno: initial import of TCP Reno congestion control (#15953)
++ drivers/sx1280: add driver for SX1280 transceiver v2 (#18033)
++ gnrc/ipv6_auto_subnets: init RPL root when adding a prefix (#17350)
++ ieee802154: add default max frame retransmissions config (#18264)
++ pkg/lwip: Add basic version of netif_get/set_opt (#18069)
++ pkg/lwip: Add netif_t struct, register all netifs (#16974)
++ pkg/nimble/autoadv: add support for ext_adv (#17774)
++ sys/iolist: add helper functions (#18044)
++ sys/net/gnrc: add gnrc_tx_sync for gnrc_sixlowpan_frag_sfr (#16090)
++ sys/net/gnrc_pktbuf_static: add use-after-free detection (#18227)
++ sys/net/netif: add convenience functions for getting/printing IPv6
+  addresses (#18161)
+* gnrc/ipv6_auto_subnets: allow to join multiple subnets (#17547)
+* gnrc_ipv6_nib: fix inconsistent ABR valid lifetime representation (#18126)
+* gnrc_netif: make auto-config of compression context optional (#17678)
+* gnrc_netif: use event loops by default to process ISR (#16748)
+* gnrc_sock: implicitly set netif if there is only a single one (#18065)
+* gnrc_sock: warn about non-zero receive timeouts with sock_async (#17691)
+* net/ble/skald: make advertising interval configurable per context (#17834)
+* net/gnrc: Fixing whitespace in the documentation (#18036)
+* net/sock/udp: designated intializer, C++ compatibility (#18035)
```

Comment:
```suggestion
* net/sock/udp: designated initializer, C++ compatibility (#18035)
```

Given that this likely is the PR title, you may want to add a (sic!) instead :)

Reply:
It's in the PR title, but I've altered a few texts already where the PR titles were not helpful in the context, so these will just be replaced.